<article class="bugdoc">
    <h2 class="bugdoc-title" id="Bug-Doc:PE22525">PE22525: E0108</h2>
    <h3>E0108</h3>
    <section id="tip">
     <h4 style="border-top: 0pt; padding: 0pt 4pt 4pt 4pt;">Bug info</h4>
                <dl>
                    <dt>BugTypeID</dt>
                    <dd>PE22525</dd>
                    <dt>Tags</dt>
                    <dd>flake8</dd>
                    <dt>Category</dt>
                    <dd>Code style and efficiency</dd>
                    <dt>Authors</dt>
                    <dd>pinpoint</dd>
                </dl>
    <h2>Message</h2>
<p>Duplicate argument name %s in function definition</p>
<h2>Description</h2>
<p>Used when an argument name is used more than once in a function or method definition.</p>
<p>This message belongs to the basic checker.</p>
<h2>Explanation</h2>
<p>Python 2.6(?) and above raises a SyntaxError for this situation when importing the module. PyLint reports this error using this message, without depending on the import actually happening.</p>
<h4>References</h4>
<section class="footnotes">
<ol>
<li id="fn-1"><p>https://www.python.org/dev/peps/pep-0008/<a href="#fnref-1" class="footnote">&#8617;</a></p></li>
</ol>
</section>

    </section>
</article>
